lib/server.js

```diff
@@ -243,14 +243,18 @@ class CoAPServer extends events.EventEmitter {
 
         if (!this._options.type) this._options.type = 'udp4'
 
+        if (this._options.reuseAddr == null) {
```

Comment:
I would have not done the == null check that way (more like !== false ... is true :-) and only false is false ... would be my comment here :-)

Reply:
Oh, very good point! I will fix it in a follow-up PR :) Thank you for going through all of these PRs by the way! :)

Reply:
Was shocked by the number :-) I'm not commenting all ... just some :-)